### PR TITLE
Add some links pointing to different parts of the create-source docs

### PR DESCRIPTION
**Pull request by quodlibetor**
CC @benesch you said you want to get pinged when I wish there had been more docs.

---
This change is [Reviewable](https://reviewable.io/reviews/materializeinc/materialize/5525)

### CODE REVIEW

**Comment by benesch** [comment]
Thanks! Seems like a job for @sploiselle.

File: doc/user/README.md [hunk 1]
```diff
@@ -110,7 +110,12 @@ each meaningfully distinct combination of connector and format), and leveraged
 Hugo partials and shortcodes to simplify page creation and resuse content across
 pages.
 
-You can find all of the `CREATE SOURCE` docs at [content/sql/create-source](content/sql/create-source/).
+* You can find all of the `CREATE SOURCE` docs at
+  [content/sql/create-source](content/sql/create-source/).
+* Each `CREATE SOURCE` doc page will call the `create-source` *layout* to provide uniform details
```
Comment (sploiselle) [inline doc/user/README.md:115]: missing word at the EOL here?

Comment (quodlibetor) [reply doc/user/README.md:115]: Ah, markdown rendering removes the newline, so the path shows up as the correct EOL:

image

Comment (quodlibetor) [reply doc/user/README.md:115]: I can join the lines if you'd prefer?

Comment (sploiselle) [reply doc/user/README.md:115]: Ohhhh, derp––so sorry. This was just my early-morning brain not able to handle the complexity; this is fine as-is

**Pull request merged**